scripts/install.sh

```diff
@@ -22,6 +22,18 @@ else
 	exit 1
 fi
 
+
+if [ "$(uname -m)" == "x86_64" ]; then
+    MACHINE="x86_64"
```

Comment:
Should we set this variable at the top of the file with `x86_64` as the default? Seems to be a prevalent pattern in the make files.